platform-service/src/unstract/platform_service/helper.py

```diff
@@ -59,12 +70,20 @@ def get_prompt_instance_from_db(
         Returns:
             _type_: _description_
         """
-        query = (
-            f"SELECT prompt_registry_id, tool_spec, "
-            f"tool_metadata, tool_property FROM "
-            f'"{organization_id}".prompt_studio_registry_promptstudioregistry x'
-            f" WHERE prompt_registry_id='{prompt_registry_id}'"
-        )
+        if check_feature_flag_status(FeatureFlag.MULTI_TENANCY_V2):
+            query = (
+                "SELECT prompt_registry_id, tool_spec, "
+                "tool_metadata, tool_property FROM "
+                f"{DBTableV2.PROMPT_STUDIO_REGISTRY} x "
+                f"WHERE prompt_registry_id='{prompt_registry_id}'"
```

Comment:
@muhammad-ali-e  dont we have to filter by org?

Reply:
I think registry id will be unique across the organizations. If else we can add conditions according to the model.

Reply:
Ah ok, this is in platform service, I hope the new model has org_id because it should be used in filtering the tools on the unstract side.